releasenotes/notes/windowsdriver20-11f39d5a55f319c0.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    On Windows, NPM driver adds support for Receive Segment Coalescing support.
+    This works around Windows bug in which, in some situations, system probe 
```

Comment:
```suggestion
    This works around a Windows bug which in some situations, system probe 
```

Reply:
@jtappa 
How about
`This works around a Windows bug which in some situations causes system probe to hang on startup`

Reply:
That works for me!